src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -71,18 +70,20 @@ class FormStartControls extends React.Component {
       this.props.prefillTransformer,
     );
 
-    const { formConfig = {} } = this.props.routes?.[1] || {};
+    const { formConfig = {} } =
+      this.props.routes?.[1] || this.props.formConfig || {};
     // Wizard status needs an intermediate value between not-started &
     // complete to prevent infinite loops in the RoutedSavableApp
     sessionStorage.setItem(
-      formConfig.wizardStorageKey || WIZARD_STATUS,
+      formConfig?.wizardStorageKey || WIZARD_STATUS,
       WIZARD_STATUS_RESTARTING,
     );
   };
 
   render() {
     // get access to the formConfig object through this route
-    const { formConfig } = this.props.routes[1];
+    const { formConfig } =
+      this.props?.routes?.[1] || this.props.formConfig || {};
```

Comment:
This change, and line 74, are the essential changes to this file. Everything else was cleanup & updating.